src/Components/Web.JS/src/Rendering/JSRootComponents.ts

```diff
@@ -121,7 +130,10 @@ export function enableJSRootComponents(
   if (manager) {
     // This will only happen in very nonstandard cases where someone has multiple hosts.
     // It's up to the developer to ensure that only one of them enables dynamic root components.
-    throw new Error('Dynamic root components have already been enabled.');
```

Comment:
I don't think this should be changed. This error should not be reported to the person calling `Blazor.rootComponents.add`, but rather to the person calling `enableJSRootComponents`. Could you revert this block of changes?

Reply:
Also, to clarify since maybe it might seem like we should also reject the promise in this case: this exception only happens on the 2nd and later call to this function, by which time the promise resolver will already have been called. So calling the promise rejecter would be a no-op, hence there's never a situation where we'd be reporting this failure to the person who called `Blazor.rootComponents.add`.